test/functional/kernel/tile/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+# Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+#
+# Generate kernel fixed tile tests for each enabled RAJA back-end.
+#
+set(TILETYPES Fixed2D)
+
+foreach( TILE_BACKEND ${KERNEL_BACKENDS} )
+  foreach( TILE_TYPE ${TILETYPES} )
+    configure_file( test-kernel-tilefixed.cpp.in
+                    test-kernel-tile-${TILE_TYPE}-${TILE_BACKEND}.cpp )
+    raja_add_test( NAME test-kernel-tile-${TILE_TYPE}-${TILE_BACKEND}
+                   SOURCES ${CMAKE_CURRENT_BINARY_DIR}/test-kernel-tile-${TILE_TYPE}-${TILE_BACKEND}.cpp )
+
+    target_include_directories(test-kernel-tile-${TILE_TYPE}-${TILE_BACKEND}.exe
+                               PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/tests)
+  endforeach()
+endforeach()
+
+unset( TILETYPES )
+
+#
+# Generate kernel dynamic tile tests for each enabled RAJA back-end.
+#
+set(TILETYPES Dynamic2D)
+
+foreach( TILE_BACKEND ${KERNEL_BACKENDS} )
+  foreach( TILE_TYPE ${TILETYPES} )
+    # Dynamic tiling not yet implemented for Cuda or Hip
+    # Removing OpenMPTarget because XLC compilation requires ~50 minutes
```

Comment:
Hi @rhornung67, would you be ok with this? OpenMPTarget builds and runs correctly (with XL), but it takes quite a while, so I've removed it from testing.

Reply:
That's OK for now.  We need to take a step back and determine a bare minimum set of tests for OpenMP target ("smoke" tests) that will be the default to build and run when OpenMP target is enabled. Then, optionally by setting a CMake variable, the entire set of tests can be built and run.  I'll set up a meeting with a subset of us to discuss this and decide how we want to do this.